### DISCUSSION TITLE
我记得B站之前有一个作者亲自演示的教程视频，今天想复习找不到了，是侵权了嘛？作者大大方便分享一下嘛。

### DISCUSSION THREAD

**Discussion by luzujiall** [windingwind/zotero-better-notes#770] [Q&A] [answered]
如题

**Comment by xbbpbb** [comment, marked as answer]
【2022.06.26 特别活动 | Zotero 文献管理与知识整理最佳实践】 https://www.bilibili.com/video/BV1Cv4y1M7BY/?share_source=copy_web&vd_source=cee8f496a34abe941a54e13fd4f1c704
